README.md

```diff
@@ -1,4 +1,6 @@
-# `reek`: code smell detection for Ruby
+![reek logo](logo/reek.text.png)
```

Comment:
Hmmm, no offense, but the way it looks right now I thought it was a layout bug...
How about scaling the graphic down and align the "Reek" literal vertically to the ruby?